DocumentedExamples/regional-mom6-forced-by-access-om2.ipynb

```diff
@@ -0,0 +1,1238 @@
+{
```

Comment:
<p>This won't work with the current version because the setup_run_directory function is written assuming that we installed with conda, not importing directly from path. With this<a href="https://github.com/COSIMA/regional-mom6/pull/177" target="_blank"> new PR</a> though it'll work fine either way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Good point! Thanks for the new PR.</p><p>But anyway we should merge this notebook only when it works with the version installed on the conda environment. So I suggest after the PR you opened is merged, we tag a new release, ask CMS to update the regional-mom6 package in the environment and see what happens.</p><p> </p>